### ISSUE TITLE
Redraw bug when closing jump lists

### ISSUE THREAD

**Issue by dgnuff** [Open-Shell/Open-Shell-Menu#109]
Steps to reproduce:

1. Open Start menu settings and select Windows 7 style
2. Drag your taskbar to the right side of the screen.
3. Click on the start button.
4. Move the mouse to hover over an item in the left column that has a jump list accessible.

Observed: The jump list opens correctly, but the search / shutdown items at the bottom do not draw correctly.

5 Move the mouse to an item in the left column that does nit have a jump list.

Observed: The jump list closes correctly, the Shut Down button now draws correctly, but the "Search Programs and Files" entry remains corrupted.

**Comment by bonzibudd** [comment]
I can't replicate this. Possibly it's a problem with the skin?
A screenshot would be useful.

**Comment by dgnuff** [comment]
Yes, it is the skin.  I normally use the Windows 8 skin, I switched to the Classic skin, and the problem went away.  Switch back, and it starts happening again.

Just for the sake of completeness, here are three screen shots that show the issue: the first is the menu open correctly, the second it with a jump list open, and the third is after closing it again.

![ScreenShot_1](https://user-images.githubusercontent.com/26607068/97811787-a1e44900-1c31-11eb-994f-a6c424dddaf9.png)
![ScreenShot_2](https://user-images.githubusercontent.com/26607068/97811791-a577d000-1c31-11eb-8fb3-2a080e0c92c4.png)
![ScreenShot_3](https://user-images.githubusercontent.com/26607068/97811793-a872c080-1c31-11eb-8521-153588923cef.png)

In order to allow you to exactly replicate my setup, I've attached a backup of my settings.

[Menu_Settings.zip](https://github.com/Open-Shell/Open-Shell-Menu/files/5471576/Menu_Settings.zip)

**Comment by bonzibudd** [comment]
Found it: The issue is triggered by disabling "Animated transitions in main menu."

All of the skins which use a "search background" have this issue from what I can tell. The lack of an animation seems to offset the bitmap.

**Comment by bonzibudd** [comment]
You can minimize this by setting the Jumplist pane width to a smaller number, but this definitely isn't ideal.
